.eslintrc.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
@murediane please, rebase with develop. `.eslintrc.json` is no longer on develop. Also, I don't understand why eslint config file is part of your PR. Please, ensure only files related to your PR show up in your commits.

Reply:
done